.github/workflows/e2e.go.workflow_dispatch.main.adversarial-verifier-binary.slsa3.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     needs: [build]
     if: needs.build.result == 'success'
     steps:
-      - uses: actions/checkout@f095bcc56b7c2baf48f3ac70d6d6782f4f553222
+      - uses: actions/checkout@96f53100ba2a5449eb71d2e6604bbcd94b9449b5
```

Comment:
nit: Need to update this to reference a release tag.

Reply:
https://github.com/slsa-framework/example-package/commit/7d18190b5538def004fb4be8d6f26969b0485155